core/src/main/java/org/mapfish/print/processor/map/CreateMapProcessor.java

```diff
@@ -511,7 +511,7 @@ private void zoomToFeatures(
         ReferencedEnvelope bounds = getFeatureBounds(clientHttpRequestFactory,
                 mapValues, context);
 
-        if (bounds != null) {
+        if (bounds != null && !bounds.isNull()) {
```

Comment:
Not relay clear, can you add a Java commend like 'test it the bounds is not null' :-)

Reply:
What is not clear in `bounds != null && !bounds.isNull()`?

Reply:
Java humor :-)